.gitignore

```diff
@@ -61,3 +61,4 @@ src/instruments/buildSrc/custom/*
 /flybywire-aircraft-a320-neo/MCDU SERVER/
 /src/mcdu-server/client/build
 !/flybywire-aircraft-a320-neo/CORS SERVER/*
+flybywire-aircraft-a320-neo/html_ui/JS/sentry-client/sentry-client.js
```

Comment:
This can be dropped again as it has been solved differently in master (it also has nothing to do with this PR).